legistar/base.py

```diff
@@ -79,6 +79,32 @@ class LegistarScraper(scrapelib.Scraper, LegistarSession):
     def __init__(self, *args, **kwargs):
         super(LegistarScraper, self).__init__(*args, **kwargs)
 
+    @property
+    def ecomment_dict(self):
```

Comment:
this seems like something that a regex would be good for, but this seems fine for now.

Reply:
https://github.com/Kronuz/esprima-python could be a good alternative.